src/component/book.js

```diff
@@ -1,37 +1,57 @@
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import Progressbar from 'react-js-progressbar';
+import { removeBook, fetchBooks } from '../redux/books/books';
 import './books.css';
 
 const Book = () => {
-  const books = useSelector((state) => state.booksReducer);
+  const books = useSelector((state) => state.bookStore);
   const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, [books]);
   return (books.map((book) => (
     <div key={book.id} className="card">
```

Comment:
- [ ] [REQUIRED] Kindly note that the content of the card is incomplete. The progress bar does not have the completed and the percentage beside it as it is in the zeplin design. Kindly ensure that you stick to all the properties of the card section (e.g color, font family, font size, font weight) because the properties you applied to your design card section are different from that on zeplin.

- [ ] [REQUIRED] The section where we have the `comment`. `remove and `edit` is implemented on your design but the stroke that separates them has been omitted. Kindly implement this as it is a core requirement.

### your design
![3](https://user-images.githubusercontent.com/58692590/180626118-b07ca704-b404-4618-9be7-81e7b37eeb15.png)

### Zeplin's design
![4](https://user-images.githubusercontent.com/58692590/180626129-6e99419d-b435-4e0f-99b6-8bae7ae9044a.png)

Reply:
- [ ] Great work on adding the completion percentage with the completed status. But kindly ensure that the completion percentage appears to the right of the circular progress bar like in the Zeplin design. Right now you have it in the center of the bar as you can see below. Kindly refer to the Zeplin design to arrange it accordingly.
![image](https://user-images.githubusercontent.com/90556221/180767355-93709ac2-9b5d-47a8-90d2-55d1a483c2db.png)